README.md

```diff
@@ -2,6 +2,13 @@
 [![Build Status](https://github.com/SchemaPlus/schema_plus_compatibility/actions/workflows/prs.yml/badge.svg)](https://github.com/SchemaPlus/schema_plus_compatibility/actions)
 [![Coverage Status](https://coveralls.io/repos/github/SchemaPlus/schema_plus_compatibility/badge.svg)](https://coveralls.io/github/SchemaPlus/schema_plus_compatibility)
 
+# Deprecation notice
+
+This gem is officially deprecated and will no longer be maintained.
```

Comment:
@urkle what about displaying a warning with this notice or using Gem::Deprecate ?